pkgs/applications/misc/stylish/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
+  pname = "stylish";
+  version = "0.0.1";
+
+  src = fetchFromGitHub {
+    owner = "thevinter";
+    repo = "styli.sh";
+    rev = "d595412a949c6cdc7e151ae0cf929aa1958aa7f1";
```

Comment:
They do not use Git tags? It should be something like `rev = version;`

Reply:
I did not find any tags [on the upstream repository](https://github.com/thevinter/styli.sh/tags)